spec/koans.js

```diff
@@ -42,39 +43,39 @@ describe('`const` is like `let` plus read-only. ', () => {
 
   describe('scalar values are read-only', () => {
     it('number are read-only', () => {
-      // const constNum = 0;
-      // constNum = 1;
+      const constNum = 0;
+      let constNum = 1;
```

Comment:
You have a DeclarationException because you can't declare twice a variable with the same name with const and let, this is the point of const. So you have to leave the second one commented out, I know the instruction is not clear though!